lib/src/method_call_handler.dart

```diff
@@ -3,26 +3,23 @@ import 'package:ably_flutter_plugin/src/generated/platformconstants.dart';
 import 'package:ably_flutter_plugin/src/impl/message.dart';
 import 'package:flutter/services.dart';
 
-
-class AblyMethodCallHandler{
-
-  AblyMethodCallHandler(MethodChannel channel){
+class AblyMethodCallHandler {
+  AblyMethodCallHandler(MethodChannel channel) {
     channel.setMethodCallHandler((call) async {
-      switch(call.method){
+      switch (call.method) {
         case PlatformMethod.authCallback:
           return await onAuthCallback(call.arguments);
       }
     });
   }
 
-  onAuthCallback(AblyMessage message) async {
+  Future onAuthCallback(AblyMessage message) async {
     ably.TokenParams tokenParams = message.message as ably.TokenParams;
     ably.Rest rest = ably.restInstances[message.handle];
-    Object callbackResponse = await rest.options.authCallback(tokenParams);
-    Future.delayed(Duration.zero, (){
+    await rest.options.authCallback(tokenParams);
+    Future.delayed(Duration.zero, () {
       rest.channels.all.forEach((c) => c.authUpdateComplete());
     });
-    return callbackResponse;
+    return;
```

Comment:
@zoechi the returned `callbackResponse` will be returned back to platform. So, we'd need this return statement to be as is.

Reply:
The function should have a proper return type then.
The type annotation in the original code is `Object` (line 21), but `AuthCallback` is defined to return `dynamic`. 

I think I suggested to not use `dynamic` at all already (except a few exceptions where it's unavoidable).
I'll leave `dynamic` here anyway for consistency.

If we actually want to to consistently not use `dynamic`, we should create an issue for this and make a PR where we get rid of it everywhere.

Reply:
Sure, let's get rid of everything `dynamic`!